tests/garage/torch/modules/test_discrete_cnn_module.py

```diff
@@ -0,0 +1,111 @@
+"""Test DiscreteCNNModule."""
+import pickle
+
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+
+from garage.torch.modules import CNNModule, DiscreteCNNModule, MLPModule
+
+
+class TestDiscreteCNNModule:
```

Comment:
can you use a fixture and a pytest style test? we have been trying to discourage new xunit-style tests

Reply:
Do you have an example? Is the difference simply that I shouldn't use a class?